daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -975,7 +960,7 @@ private[lf] final class Compiler(
     // Translates 'create Foo with <params>' into:
     // CreateDefRef(tmplId) = \ <tmplArg> <token> ->
     //   let _ = checkPreCondition(tmplId, <tmplArg>)
-    //   in $create <tmplArg> [tmpl.agreementText] [tmpl.signatories] [tmpl.observers] [tmpl.key]
+    //   in $create <tmplArg> [tmpl.signatories] [tmpl.observers] [tmpl.key]
```

Comment:
The original comment is actually wrong.
```suggestion
    //   in $create <tmplArg>
```